src/routes/bets/+page.server.ts

```diff
@@ -56,34 +45,96 @@ export const load: PageServerLoad = async ({ locals: { supabase }, parent }) =>
         // Form data
 		const formData = await request.formData();
 		const bet = formData.get('bet')?.toString();
 		const value = formData.get('value')?.toString();
 
-        // Server data (can we get from PAGE instead of re-loading?)
+        // Get from hooks
 		const session = await getSession();
         const season = await getSeason();
 
 		if (session && season && bet && value) {
-            // Using a GENERIC INSERT where id is optional
             const betForm: TablesInsert<'bets'> = {
                 user_id: session.user.id,
                 season_id: season.id,
                 bet: bet,
                 value: parseInt(value),
             }
 
-            const { error: error } = await supabase.from('bets').insert(betForm);
+            const { error: insertError } = await supabase.from('bets').insert(betForm);
+
+			if (insertError) {
+                throw error(500, {
+                    message: insertError.message,
+                    devHelper: '/bets inserting a bet'
+                });
+			}
+        }
+        
+        throw redirect(302, '/bets');
+	},
+    addBetAgainst: async ({ request, locals: { supabase, getSession, getSeason } }) => {
+        // Form data
+		const formData = await request.formData();
+		const bet_id = formData.get('bet_id')?.toString();
+
+        // Get from hooks
+		const session = await getSession();
+        const season = await getSeason();
+
+		if (session && season && bet_id) {
+            const betAgainstForm: TablesInsert<'bets_against'> = {
+                bet_id: parseInt(bet_id),
+                user_id: session.user.id,
+            }
+            const { error: insertError } = await supabase.from('bets_against').insert(betAgainstForm);
+
+			if (insertError) {
+                throw error(500, {
+                    message: insertError.message,
+                    devHelper: '/bets inserting a bet_against'
+                });
+			}
+        }
+        
+        throw redirect(302, '/bets');
+	},
+    removeBetAgainst: async ({ locals: { supabase, getSession, getSeason } }) => {
+        // Get from hooks
+		const session = await getSession();
+        const season = await getSeason();
+
+		if (session && season) {
+            const { error: deleteError } = await supabase.from('bets_against').delete().eq('user_id', session.user.id);
+
+			if (deleteError) {
+                throw error(500, {
+                    message: deleteError.message,
+                    devHelper: '/bets deleting a bet_against'
+                });
+			}
+        }
+
+        throw redirect(302, '/bets');
+	},
+    removeBet: async ({ locals: { supabase, getSession, getSeason } }) => {
+        // Get from hooks
+		const session = await getSession();
+        const season = await getSeason();
+
+		if (session && season) {
+            const { error: deleteError } = await supabase.from('bets').delete().eq('user_id', session.user.id);
```

Comment:
seeing as we use the session object here i don't think its a big deal, but keeping RLS on for bets might be an idea cause you could wreak some havoc with these actions somehow if not

(not that hacking is a legitimate thing we care about)